docs/publish/pom.xml

```diff
@@ -130,7 +130,7 @@
                                         <artifactItem>
                                             <groupId>org.glassfish.docs</groupId>
                                             <artifactId>distribution</artifactId>
-                                            <version>${glassfish.version.7x}</version>
+                                            <version>${glassfish.version.7x.artifact}</version>
```

Comment:
Does this change affect the release process?

Reply:
No, this change is only to reflect what is happening right now. Currently, we don't publish the docs artifact as part of the release process, the docs are only published daily for the snapshot version:
* There's only 7.0.0 version in Maven Central: https://search.maven.org/search?q=g:org.glassfish.docs%20AND%20a:distribution
* There's only 7.0.0 version and snapshot versions in Jakarta Maven repo: https://jakarta.oss.sonatype.org/content/groups/staging/org/glassfish/docs/distribution/

So we need to download the snapshot artifact instead of the final artifact. The rest is the same as before.

If we published the docs artifact as part of the release process, this wouldn't be necessary and we could publish the docs from the ${glassfish.version.7x} version.